msi/GitProduct.wxs

```diff
@@ -15,7 +15,7 @@
     <Package InstallScope='perMachine' Compressed='yes' Keywords='Git' />
     <swid:Tag Regid='git-for-windows.github.io' />
 
-    <MajorUpgrade DowngradeErrorMessage='A newer version of [ProductName] is already installed.' />
+    <MajorUpgrade DowngradeErrorMessage='A newer version of [ProductName] is already installed.' Schedule="afterInstallInitialize" />
```

Comment:
I'd rather have this as a separate commit (or the original commit fixed, if it is part of this here Pull Request).